static/sass/components/header.scss

```diff
@@ -1,11 +1,6 @@
-.navbar .nav-item.active .nav-link {
-    color: $themeColor
-}
-
 .page-title-short {
-    padding: 0;
-    background: grey;
-    padding: 120px 0 30px 0;
+    background: #464B54;
```

Comment:
Can we move this to a variable? I've seen it being used [twice](https://github.com/saucelabs/saucelabs.github.io/pull/3/files#diff-d41313d4b8934963feaaa34c1af03fe4R219).

Reply:
Yeah, but the color is the same, but where the style is used has two different purposes, one is to be the background of the projects page, and the second is to be the active link color of a navigated content page.

So, it is the same value but if we move this to a variable, it would need to be two different variables to reflect the purpose. What do you think?

Reply:
Let's declare it as followed then:

```sass
$darkGray: #464B54;

$projectPageBackgroundColor: $darkGray;
$activeLinkColor: $darkGray; 
```